### ISSUE TITLE
Nukeops player requirement too high (35 ready players)

### ISSUE THREAD

**Issue by EkrellX** [DeltaV-Station/Delta-v#1403]
## Description
In the code, we can see that 35 players at minimum are required for the NukeOps round to enter the rotation. While not an issue on this level, it becomes one when you realize it's READY players we are talking about. I believe it was an overestimation since 90% of rounds outside of 70-80 players on the server will not meet this requirement. Most people from what I've gathered wish to see what roles are taken first, before entering the round (or are purposefully not getting ready to avoid NukeOps). This essentially removes NukeOps from the game outside of 70+ players lobbies. Add the 15% chance on top of that and you get 1 NukeOps round (2 if you are really lucky) max once a day (at least during European Time). You may argue that low-pop NukeOps are just boring to play against, and yeah - that's true, but I guarantee you that lowering the requirement to 30 or 25 will still make lobbies of 40-ish people safe and sound. Arguments that SEC won't be able to handle the situation also fall apart, since you can just get unlucky and get no SEC in max player lobbies too.

In conclusion, NukeOps should be lowered to 30 or 25 minPlayers instead of 35. Alternatively swapped up to more if you somehow switch it to actual players on the SERVER rather than READY at the start.

**Reproduction**
Enter a game with less than 60-70 people. Cry.

**Screenshots**
![cry](https://github.com/DeltaV-Station/Delta-v/assets/113257334/96376dfe-d8b6-441c-9c3b-2bb471b40ba5)

**Additional context**
On the screenshot, we can see an example of **68** players lobby (Basically 85ish% server full) not managing to meet the requirement for NukeOps. Not the first time it has happened anyways.
